app/src/main/java/com/example/calculatorkt/mvp/view/CalculatorView.kt

```diff
@@ -1,13 +1,55 @@
 package com.example.calculatorkt.mvp.view
 
 import android.app.Activity
+import com.example.calculatorkt.R
 import com.example.calculatorkt.databinding.ActivityMainBinding
 import com.example.calculatorkt.mvp.CalculatorContract
+import com.example.calculatorkt.util.Constants.EMPTY
+import com.google.android.material.snackbar.Snackbar
 
 class CalculatorView(activity: Activity, private val binding: ActivityMainBinding) :
     ActivityView(activity), CalculatorContract.View {
 
-    override fun showOperationValue(operation: String) {
+
+    override fun showOperationValue(operation: String?) {
```

Comment:
the type of this operation is correct can be null?

Reply:
No, it cannot be null. Solved!